internal/web/handlers/json.go

```diff
@@ -0,0 +1,70 @@
+package handlers
+
+import (
+	"encoding/json"
+	"net/http"
+	"net/url"
+
+	"github.com/shulganew/shear.git/internal/config"
+	"github.com/shulganew/shear.git/internal/service"
+	"go.uber.org/zap"
+)
+
+type RequestJSON struct {
+	LongURL string `json:"url"`
+}
```

Comment:
json лишнее т.к. это структура, а анотация, тегирование может много чего еще - например если добавить db: ... эта структура уже будет парсится при добавлении или чтении из базы данных. Так что лучше просто Request/Response - а внутри максимально повторять имена чтобы не было путаницы (хотя LongURL даже понятней)
```suggestion
type Request struct {
	URL string `json:"url"`
}
```